src/bookshelf/rebar.config

```diff
@@ -2,46 +2,52 @@
     {cf, "",
-        {git, "git://github.com/project-fifo/cf.git",                 {branch, "master"}}},
+        {git, "git://github.com/project-fifo/cf.git",                   {branch, "master"}}},
     {chef_secrets, ".*",
-        {git, "git://github.com/chef/chef_secrets.git",               {branch, "master"}}},
+        {git, "git://github.com/chef/chef_secrets.git",                 {branch, "master"}}},
     {envy, ".*",
-        {git, "git://github.com/markan/envy.git",                     {branch, "master"}}},
+        {git, "git://github.com/markan/envy.git",                       {branch, "master"}}},
     {eper, ".*",
-        {git, "git://github.com/massemanet/eper.git",                 {branch, "master"}}},
+        {git, "git://github.com/massemanet/eper.git",                   {branch, "master"}}},
     {erlsom, ".*",
-        {git, "git://github.com/chef/erlsom.git",                     {branch, "integer_long_string_probs2"}}},
+        {git, "git://github.com/chef/erlsom.git",                       {branch, "integer_long_string_probs2"}}},
     {erlware_commons, ".*",
-        {git, "https://github.com/chef/erlware_commons.git",          {branch, "lbaker/fix_for_ftmap"}}},    
+        {git, "git://github.com/chef/erlware_commons.git",              {branch, "lbaker/fix_for_ftmap"}}},
     {iso8601, ".*",
-         {git, "git://github.com/erlsci/iso8601.git",                 {tag, "1.2.3"}}},
+         {git, "git://github.com/erlsci/iso8601.git",                   {tag,    "1.2.3"}}},
     {meck, ".*",
-         {git, "git://github.com/eproxus/meck.git",                   {tag, "0.8.4"}}},    
+        %{git, "git://github.com/eproxus/meck.git",                      {tag,    "0.8.4"}}},
+        {git, "git://github.com/eproxus/meck.git",                      {branch, "master"}}},
     {mini_s3, ".*",
-        {git, "git://github.com/chef/mini_s3.git",                    {branch, "master"}}},
+        {git, "git://github.com/chef/mini_s3.git",                      {branch, "master"}}},
     {mixer, ".*",
-        {git, "git://github.com/chef/mixer.git",                      {tag, "0.1.1"}}},    
+        {git, "git://github.com/chef/mixer.git",                        {tag,    "0.1.1"}}},
     {mochiweb, ".*",
-        {git, "git://github.com/mochi/mochiweb.git",                  {tag, "v2.12.2"}}},
+        %{git, "git://github.com/mochi/mochiweb.git",                    {tag,    "v2.12.2"}}},
+        {git, "git://github.com/mochi/mochiweb.git",                    {branch, "master"}}},
     {observer_cli, ".*",
-        {git, "git://github.com/zhongwencool/observer_cli.git",       {branch, "master"}}},
+        {git, "git://github.com/zhongwencool/observer_cli.git",         {branch, "master"}}},
     {opscoderl_wm, ".*",
-        {git, "git://github.com/chef/opscoderl_wm.git",               {branch, "master"}}},
+        {git, "git://github.com/chef/opscoderl_wm.git",                 {branch, "lbaker/update-erlang22"}}},
```

Comment:
We think this can be `master` now since we've merged the fixes upstream.

Reply:
Roger that.

Reply:
Fixed.